apps/server/src/server.ts

```diff
@@ -2,27 +2,30 @@ import express, { json, urlencoded } from 'express';
 import helmet from 'helmet';
 import { serve, setup } from 'swagger-ui-express';
 
-import { getServerConfig } from './config/server';
-import swaggerDoc from './config/swagger';
-import dictionaryRouter from './routes/dictionary';
-import pingRouter from './routes/ping';
+import { errorHandler, manager } from 'common';
+import { defaultAppConfig, getServerConfig } from './config/server.js';
+import swaggerDoc from './config/swagger.js';
+import pingRouter from './routes/ping.js';
 
 const serverConfig = getServerConfig();
 
+const lyricManager = manager(defaultAppConfig);
```

Comment:
what does "manager" mean in this context?

Reply:
is a function that creates a manager, it returns an instance to access all the routers, services and other things provided by lyric lib. I guess it should be renamed as `createManager`?

Reply:
oh I don't have an issue with it being called this or that (thought `createWhatever` does seem appropriate for that instance creator). I'm just curious about the word "manager" itself, is that meant as a synonym for `router`?

e.g. would this work based on what you're trying to do:`const lyricRouter = createRouter(defaultConfigs);`?

Reply:
ah, I see. thanks for explaining: like a Lyric `provider`. that makes sense...  I was going to ask if those "specific functions" rely on the configurations you pass into the "manager" creator, but I can see now how those are all connected.

semantically speaking "manager" is ambiguous and that's why I'm asking about it.

Reply:
renamed to `provider`